archunit/src/test/java/com/tngtech/archunit/lang/ArchRuleTest.java

```diff
@@ -250,6 +279,14 @@ public void check(JavaClass item, ConditionEvents events) {
                 }
             };
 
+    private static final ArchCondition<JavaClass> ALWAYS_BE_VALID =
+            new ArchCondition<JavaClass>("always be valid") {
+                @Override
+                public void check(JavaClass item, ConditionEvents events) {
+                    events.add(new SimpleConditionEvent(item, true, "I'm valid"));
```

Comment:
You don't need to add an event here :wink: The condition will also pass if there is no event at all...